glslc/src/file_compiler.cc

```diff
@@ -177,6 +177,24 @@ void FileCompiler::OutputMessages() {
   shaderc_util::OutputMessages(&std::cerr, total_warnings_, total_errors_);
 }
 
+namespace {
+std::string GetBaseNameFromAbsolutePath(const std::string& path) {
+  if (path.empty()) return "";
+  // Unix-like OS: /path/to/file
+  if (path.front() == '/') return path.substr(path.find_last_of('/'));
+  // Windows: \\server\user\file
+  if (path.size() > 1 && path[0] == '\\' && path[1] == '\\') {
+    return path.substr(path.find_last_of('\\'));
+  }
+  // Windows: X:\path\to\file
+  if (path.size() > 2 && ::isalpha(path[0]) && path[1] == ':' &&
+      path[2] == '\\') {
+    return path.substr(path.find_last_of('\\'));
+  }
+  return "";
+}
+} // anonymous namespace
```

Comment:
Looks like you didn't run clang-format?
That normally puts two spaces before the namespace comment.